components/ingest-service/backend/elastic/elastic.go

```diff
@@ -241,53 +241,70 @@ func (es *Backend) createBulkUpdateRequestToIndexWithID(
 
 // InitializeStore runs the necessary initialization processes to make elasticsearch usable
 // in particular it creates the indexes and aliases for documents to be added
-func (es *Backend) InitializeStore(ctx context.Context) {
+func (es *Backend) InitializeStore(ctx context.Context) error {
 	if !es.initialized {
 		for _, esMap := range mappings.AllMappings {
 			if esMap.Timeseries {
-				es.createTemplate(ctx, esMap.Index, esMap.Mapping)
+				err := es.createTemplate(ctx, esMap.Index, esMap.Mapping)
+				if err != nil {
+					return err
+				}
 			} else {
-				es.createOrUpdateStore(ctx, esMap)
-				es.createStoreAliasIfNotExists(ctx, esMap.Alias, esMap.Index)
+				err := es.createOrUpdateStore(ctx, esMap)
```

Comment:
An error will be returned here for node-state mapping updates. This would have prevented the 'projects' field problem.